* Fri May 11 2007 Adam Jackson <ajax@redhat.com> 1.3.0.0-5
- xserver-1.3.0-fbdevhw-magic-numbers.patch: If the fbdev driver claims to
  have a zero pixel clock, believe it.  Fixes Xen paravirt. (#238451)

Might fix #422430

--- xorg-server-1.3.0.0/./hw/xfree86/fbdevhw/fbdevhw.c.jx	2007-03-26 23:32:36.000000000 -0400
+++ xorg-server-1.3.0.0/./hw/xfree86/fbdevhw/fbdevhw.c	2007-05-11 15:10:05.000000000 -0400
@@ -258,7 +258,7 @@
 static void
 fbdev2xfree_timing(struct fb_var_screeninfo *var, DisplayModePtr mode)
 {
-	mode->Clock = var->pixclock ? 1000000000/var->pixclock : 28000000;
+	mode->Clock = var->pixclock ? 1000000000/var->pixclock : 0;
 	mode->HDisplay = var->xres;
 	mode->HSyncStart = mode->HDisplay+var->right_margin;
 	mode->HSyncEnd = mode->HSyncStart+var->hsync_len;
